packages/1-framework/3-tooling/migration/src/space-layout.ts

```
@@ -13,12 +13,19 @@ export { APP_SPACE_ID };
  */
 export type ValidSpaceId = string & { readonly __brand: 'ValidSpaceId' };
 
+/**
+ * Branded string carrying a compile-time guarantee that the value has
+ * been validated by {@link assertValidSpaceId}. Downstream filesystem
+ * helpers (e.g. {@link spaceMigrationDirectory}) accept this type to
+ * make "validated" tracking visible at the type level rather than
+ * relying purely on a runtime check.
+ */
+export type ValidSpaceId = string & { readonly __brand: 'ValidSpaceId' };
```

Biome (2.4.14)

[error] 'ValidSpaceId' is redeclared in the same scope.

(lint/suspicious/noRedeclare)